Tests/SentryTests/Integrations/OutOfMemory/SentryOutOfMemoryTrackerTests.swift

```diff
@@ -9,9 +9,10 @@ class SentryOutOfMemoryTrackerTests: NotificationCenterTestCase {
     private class Fixture {
         
         let options: Options
-        let client: TestClient!
+        let client: TestClient
         let crashWrapper: TestSentryCrashWrapper
-        let fileManager: SentryFileManager
+        lazy var mockFileManager = try! TestFileManager(options: options, andCurrentDateProvider: currentDate)
+        lazy var realFileManager = try! SentryFileManager(options: options, andCurrentDateProvider: currentDate, dispatchQueueWrapper: dispatchQueue)
```

Comment:
This one was more interesting. A couple of the test cases did need a real file manager to manipulate some files on disk.